cmake/modules/VersionInfo.cmake

```diff
@@ -13,13 +14,18 @@ IF(GIT_FOUND AND NOT FORCE_VERSION)
 		LIST(GET TAG_LIST 0 FORCE_VERSION)
 		STRING(REPLACE "v" "" FORCE_VERSION "${FORCE_VERSION}")
 	ENDIF()
-	IF(TAG_LIST_LENGTH EQUAL 2)
+	IF(TAG_LIST_LENGTH LESSL 2)
```

Comment:
```suggestion
	IF(TAG_LIST_LENGTH LESS 2)
```

Reply:
I'm already aware of the typo (build fails).  Thanks :D